src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,98 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.10.0dev2 (08/10/2018)
```

Comment:
Both this and the previous two releases should have an extra dot: `1.10.0.dev2`.

Reply:
Fixed